notebooks/05-visualize-distributions-boxplots.ipynb

```diff
@@ -0,0 +1,353 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        x2 = ridge(cat, -(np.exp(log_dens)))
</pre><p>Maybe combine rename <code>ridge</code> to <code>violin</code> and have it return both <code>x1</code> and <code>x2</code> then:</p><pre class="ql-syntax" spellcheck="false">x1, x2 = violin(cat, np.exp(log_dens))
</pre><p>Also <code>log_density</code> is probably a better name (I know the other example used the abbreviated name but I'm not sure why) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>